content/logs/faq/_index.md

```diff
@@ -12,4 +12,8 @@ private: true
     {{< nextlink href="/logs/faq/i-have-a-custom-log-file-with-heightened-read-permissions" >}}I have a custom log file with heightened read-permissions; how do I send it to Datadog?{{< /nextlink >}}
     {{< nextlink href="/logs/faq/log-parsing-best-practice" >}}Log Parsing - Best Practice{{< /nextlink >}}
     {{< nextlink href="/logs/faq/how-to-investigate-a-log-parsing-issue" >}}How to investigate a log parsing issue?{{< /nextlink >}}
+    {{< nextlink href="/logs/faq/why-my-logs-do-not-have-the-expected-timestamp" >}}Why my logs do not have the expected timestamp?{{< /nextlink >}}
```

Comment:
I would change the phrasing from `Why my logs do not have the expected timestamp?` to `Why do my logs not have the expected timestamp?`.

The url would then be `/logs/faq/why-do-my-logs-not-have-the-expected-timestamp`.

Reply:
@l0k0ms could you do the change?

Reply:
👍